.ci/pytorch/python_doc_push_script.sh

```diff
@@ -123,7 +123,6 @@ mv "$pt_checkout/docs/build/html" "$install_path"
 # generated source files.
 # NB: the following only works on gnu sed. The sed shipped with mac os is different.
 # One can `brew install gnu-sed` on a mac and then use "gsed" instead of "sed".
-find "$install_path/_modules" -name "*.html" -print0 | xargs -0 sed -i '/<head>/a \ \ <meta name="robots" content="noindex">'
```

Comment:
Why do you want to remove this? Docs should still be removed from indexing, shouldn't they?

Reply:
This: <meta name="robots" content="noindex">' only to the files _modules. These are the files like this: https://pytorch.org/docs/2.6/_modules/torch.html#is_tensor - they were generated by the sphinx.viewcode extension that we replaced with the linkcode extension that goes directly to github source. Therefore those files won't be part of the build anymore.